contracts/NFTStorefront.cdc

```diff
@@ -59,9 +59,12 @@ pub contract NFTStorefront {
         storefrontAddress: Address,
         listingResourceID: UInt64,
         nftType: Type,
+        nftUUID: UInt64, 
         nftID: UInt64,
         ftVaultType: Type,
-        price: UFix64
+        price: UFix64,
```

Comment:
should there be a currency related to price?  BEs and FEs are using a PriceV2: {currency: string, value: Float(aka UFIx64?)} for exchange. We support trade tickets on TS but potentially a customer could list also list an nft for trade tickets or whatever future currencies we or others create.

Reply:
ftVaultType is currency.